### DISCUSSION TITLE
Prioritise custom emoji?

### DISCUSSION THREAD

**Discussion by philipaarseth** [missive/emoji-mart#629] [Q&A] [answered]
Hi,

Couldn't find anything on this, but is it possible to always prioritise your custom emojis over the default ones?

**Comment by EtienneLem** [comment, marked as answer]
Yep, you can change the order of the categories (including your custom categories) with the `categories` array option, i.e.:

```js
categories: ['frequent', 'my_custom_category_id', 'people', 'nature', 'foods', 'activity', 'places', 'objects', 'symbols', 'flags']
```
